src/main/java/io/jcoder/tutorials/ch11/CatchMultipleTypes/CatchMultipleTypesApp.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2018 - JCoder Ltd
```

Comment:
minor - please update the Copyright year :)

Reply:
Fixed it and pushed the corrected version to my branch.